.github/workflows/debug_build.yml

```
@@ -0,0 +1,363 @@
+# /********************************************************************************
+# * Copyright (c) 2024 Contributors to the Eclipse Foundation
+# *
+# * See the NOTICE file(s) distributed with this work for additional
+# * information regarding copyright ownership.
+# *
+# * This program and the accompanying materials are made available under the
+# * terms of the Apache License 2.0 which is available at
+# * http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * SPDX-License-Identifier: Apache-2.0
+# ********************************************************************************/
+
+name: Debug build databroker
+
+on:
+  push:
+    branches: [ main]
+  pull_request:
+  workflow_call:
+    secrets:
+      QUAY_IO_TOKEN:
+        required: true
+      QUAY_IO_USERNAME:
+        required: true
+  workflow_dispatch:
+
+# suffix to avoid cancellation when running from release workflow
+concurrency:
+      group: ${{ github.ref }}-${{ github.workflow }}-databroker
```

please change this to databroker-debug
```suggestion
      group: ${{ github.ref }}-${{ github.workflow }}-databroker-debug
```